commit f85958a74d9e26e7e43a5d3e65d4d77ed47f55ea
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Sep 3 04:58:00 2017 +0200

    hurd: Fix libc link
    
    This follows db3d848 ('Build divdi3 only for architecture that required
    it').
    
            * sysdeps/mach/hurd/i386/Makefile
            [$(subdir) = csu] (sysdep_routines): New rule: divdi3 object.
            [$(subdir) = csu] (sysdep-only-routines): Likewise.
            [$(subdir) = csu] (CFLAGS-divdi3.c): Likewise.

diff --git a/sysdeps/mach/hurd/i386/Makefile b/sysdeps/mach/hurd/i386/Makefile
index 5f988097c2..938d3b046d 100644
--- a/sysdeps/mach/hurd/i386/Makefile
+++ b/sysdeps/mach/hurd/i386/Makefile
@@ -6,3 +6,11 @@ endif
 ifeq ($(subdir),debug)
 gen-as-const-headers += signal-defines.sym
 endif
+
+ifeq ($(subdir),csu)
+ifeq (yes,$(build-shared))
+sysdep_routines += divdi3
+shared-only-routines += divdi3
+CPPFLAGS-divdi3.c = -Din_divdi3_c
+endif
+endif
